js/main.js

```diff
@@ -1,2 +1,110 @@
 $(function () {
+
+  let playerMove
+  let botMove
+  let playerScore = 0
+  let botScore = 0
+
+  function generateBotMove (){
+    const number = Math.floor(Math.random() * 3 + 1)
+    let move
+
+    console.log(number)
+
+    if (number === 1){
+      move = "rock"
+    } else if (number === 2){
+      move = "paper"
+    } else move = "scissors"
+
+    return move
```

Comment:
Great use of the built in Math methods with if/else statements to generate a random bot choice!